### DISCUSSION TITLE
Nightly releases

### DISCUSSION THREAD

**Discussion by ttytm** [webui-dev/webui#194] [General]
This is about adding automated "nightly" releases. Until now I didn't follow through on that idea as I thought extending the current action to create a release after every commit would result in much noise. E.g. nightly releases that are announced are e.g. [neovim](https://github.com/neovim/neovim/releases) and [logseq](https://github.com/logseq/logseq/releases) - those are not automated on every commit.

BUT the nightly releases don't have to be "new" releases. So re-thinking and re-checking, we could create ONE nightly release and keep updating it, like e.g. [kitty](https://github.com/kovidgoyal/kitty/releases) does it. This would allow to use latest commit pre-builds in scripts and solve the problem discussed in https://github.com/webui-dev/v-webui/issues/59 for V and for other wrappers to follow.

What do you guys think about this?

**Comment by hassandraga** [comment]
Great idea, we should apply this. And I suggest the one nightly release and keep updating it.
Thank you @ttytm.
